<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Catapulte</class>
 <widget class="QMainWindow" name="Catapulte">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>696</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Catapulte</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="Hlayout_">
      <item>
       <widget class="QOpenGLWidget" name="SceneGL_">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>550</width>
          <height>425</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="Vlayout_">
        <item>
         <widget class="QLabel" name="Frame_">
          <property name="minimumSize">
           <size>
            <width>274</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>No image from camera!</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="Vlayout2_">
          <item>
           <widget class="QPushButton" name="Start_Button_">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="Vlayout3_">
            <item>
             <widget class="QPushButton" name="Capture_Button_">
              <property name="text">
               <string>Capture</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ReStart_Button_">
              <property name="text">
               <string>ReStart</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="Hlayout3_">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="Fire_Button_">
                <property name="text">
                 <string>Fire!!</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="Replay_Button_">
                <property name="text">
                 <string>Replay</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="Hlayout2_">
              <item>
               <layout class="QVBoxLayout" name="Vlayout4_">
                <item>
                 <widget class="QLabel" name="s_runtime_">
                  <property name="text">
                   <string>Runtime:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="s_target_time_">
                  <property name="text">
                   <string>Current Target:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="s_scores_">
                  <property name="text">
                   <string>Scores:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="Vlayout5_">
                <item>
                 <widget class="QLabel" name="v_runtime_">
                  <property name="text">
                   <string>00:00:00</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="v_target_time_">
                  <property name="text">
                   <string>00:00:00</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="v_scores_">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="Score_Table_">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string>Num</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Player</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Aimming Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Score</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
